### DISCUSSION TITLE
Welcome to KissLog.Sdk Discussions!

### DISCUSSION THREAD

**Discussion by catalingavan** [catalingavan/KissLog-net#40] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marcosmarchette** [comment]
Hi Guys,  everything is fine?
I'm trying to install KissLOg on the local machine
I did all the steps as shown on the website.
I am installing in IIS
However, when the installation is finished, it does not run.


"Cannot access this siteCheck my.kisslog-backend.com for a typo.
If the address is correct, try running Windows Network Diagnostics."
DNS_PROBE_FINISHED_NXDOMAIN

could you help me


Thanks

Marcos P. Marchette

**Reply by catalingavan** [reply]
Hi Marcos,

my.kisslog-backend.com is added just as an example.
Your KissLog should be accessible via the http://localhost:[port], where [port] is the port number of your IIS Bindings settings

Let me know if this image helps.

![image](https://user-images.githubusercontent.com/39127098/174316883-4a25bdb0-2554-4139-88f5-fa8bb7f757c0.png)

**Reply by marcosmarchette** [reply]
Thank you so much Catalin for your reply
I made the change here, but without success.

I put the website application KissLog.Backend on port 1500 and the KIssLog.Fronten on port 1501

I called the application via the URL:

"KissLogBackendUrl": "http://my.kisslog-backend.com";
"KissLogFrontendUrl": "http://my.kisslog-frontend.com"

And the error below occurs:

Cannot access this siteCheck my.kisslog-backend.com for a typo.
If the address is correct, try running Windows Network Diagnostics.
DNS_PROBE_FINISHED_NXDOMAIN

Even by localhost

NOTE: Make the correct settings in the .json files of both applications
I checked if it could be something on the network here on my internet but everything is ok



![KissLogBackend](https://user-images.githubusercontent.com/6231794/174328296-b63f62c8-2a3b-4422-ad14-5bcc528d6001.png)
s

**Comment by catalingavan** [comment]
Hi Marcos,

If you open the browser and navigate to http://localhost:1500/, what do you see? can you share a print-screen?

Thanks

**Reply by marcosmarchette** [reply]
The error appears below:
![image](https://user-images.githubusercontent.com/6231794/174330074-89a9b291-775f-4cf5-8114-e71e3370f50b.png)

**Comment by catalingavan** [comment]
Ok, that’s at least something new.
Go to the kisslog.backend website folder and check if you have a Logs folder.
If yes, can you share the file contents?

**Reply by marcosmarchette** [reply]
Yes, I have the folder but it has no content

**Reply by marcosmarchette** [reply]
I don't understand this problem very well because I followed the steps correctly according to the installation file
https://kisslog.net/Docs/on-premises.installation-guide.html

**Reply by catalingavan** [reply]
Ok. Let’s try this:
Open web.config file and set stdoutLogEnabled="true".
Restart the IIS application and try again. This time you should see something in Logs.

**Reply by marcosmarchette** [reply]
I've already done this configuration before.
This is also one of the reasons why I'm not understanding why this error

**Reply by marcosmarchette** [reply]
How is your file structure in the KissLog.Backend and KissLog.Frontend folders?

**Reply by catalingavan** [reply]
Ok. If you have stdoutLogEnabled=true and the Logs\ folder is empty, this is most probably caused because the IIS Application Pool Identity does not have permissions to write to files.
It is a bit difficult to troubleshoot using this chat.
Do you have a Skype account, where we can have a call with share screen?

**Reply by marcosmarchette** [reply]
I don't have Skype...Only Teams...Could it be?

**Reply by marcosmarchette** [reply]
https://teams.microsoft.com/l/meetup-join/19:meeting_ODM2MzMyNDItN2Q3Ny00ZmU3LWE2MGItNzZiYTViYmM2ODU4@thread.v2/0?context=%7B%22Tid%22:%225b60f289-2d18-4eea-98fa-ec3842ebe51c%22,%22Oid%22:%22735bceb8-7554-414e-8159-2057be5b9ca7%22%7D

**Reply by marcosmarchette** [reply]
Catalin Gavan
Thank's for your time
I'm sorry if I couldn't express myself better.
I will restart my computer here and do the configuration from scratch
Thanks again

**Reply by catalingavan** [reply]
Hi Marcos,
Sorry I couldn't help you too much.
The problem is most likely caused by IIS Application Pool Identity permissions, because the `stdoutLogEnabled="true"` does not write any files at all.
The first goal to achieve is to get some files under the `logs\` folder.

Please have a look over this link:
https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/logging-and-diagnostics?view=aspnetcore-6.0

Additionally, double check that you have ASP.NET Core Runtime 3.1 installed.
You can download it from here: https://dotnet.microsoft.com/en-us/download/dotnet/3.1

Let me know if you could get any logs in the `logs\` folder.

Thanks

**Reply by marcosmarchette** [reply]
Hi Catalin
How are you?

Thank you for your help
Thank you very much
I was searching the web and I managed to solve the problem.

I downloaded and ran the installation of this package:
ASP.NET Core 6.0 Runtime (v6.0.6) - Windows Hosting Bundle Installer

Link below:
https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-aspnetcore-6.0.6-windows-hosting-bundle-installer

Done that I got the return in the attached image

Here's the log in case anyone has the same problem.

Again.
Thanks for your help

God bless you
![image](https://user-images.githubusercontent.com/6231794/174455843-53ed889b-7df4-4a39-9949-15ababce290b.png)
![image](https://user-images.githubusercontent.com/6231794/174457821-2bf37ffb-b10c-406c-aca9-74eedf6b9314.png)

**Reply by catalingavan** [reply]
Hi Marcos, that is really nice, well done! Happy that it finally works!
